storage/ndb/rest-server/rest-api-server/internal/config/init_defaults.go

```diff
@@ -27,6 +27,15 @@ import (
 var globalConfig AllConfigs
 var mutex sync.Mutex
 
+/*
```

Comment:
Not sure I followed the logic in the comment, I interpret it as
if no ENV we don't check CLI argument